### PR TITLE
Add short identifiers to sections of the `Help centre` articles

### LINKED ISSUE #7498: Add identifier to `Help_centre/Account#sign-in` for the e-mail recovery link
[Context](https://github.com/ppy/osu-web/issues/8969) and [related PR](https://github.com/ppy/osu-web/pull/8986).

When users need to verify themselves via a code, they can choose to get more information about e-mail recovery in the `Help_centre/Account` article. The above mentioned PR set (hard coded) this link to `Help_centre/Account#sign-in`.

This will not work if the title of this chapter is something else than `sign-in`. It links to the article, yes, but not to the chapter. So, there needs to be an identifier to the `sign-in` chapter of `Help_centre/Account`, such that this chapter can be redirected to correctly via the e-mail recovery link in every language (it works when the language is set to English but not otherwise).

Opening this for visibility.

### PR DESCRIPTION
**Pull request by TicClick**
resolves #7498

more generally, it provides links that point to the same section of the article regardless of the translation. I attempted to choose identifiers that are descriptive enough, but you are free to contest them. https://github.com/ppy/osu-wiki/commit/f1811e96940b9a1f81daf1d8a844e330cbdfe7ef copied to the translations using VSCode's multi cursor mode, so there should be no mismatches or other issues.

if you have any official documentation pointing to the help centre, or links that are built into the client, or the website, now would be a good time to update them accordingly.

`SKIP_OUTDATED_CHECK` and `SKIP_WIKILINK_CHECK` because some articles have different issues

### CODE REVIEW

File: wiki/Help_centre/Tournament_bans/en.md [hunk 1]
```diff
@@ -30,19 +30,19 @@ This includes situations like:
...
 
-## Can I appeal a tournament ban?
+## Can I appeal a tournament ban? {#appeal}
 
 Whether temporary or permanent, **tournament bans cannot be appealed**.
 
 In rare situations, the [account support team](/wiki/People/The_Team/Account_support_team) may review individual cases at their discretion and remove or reapply appropriate punishments to ensure they remain consistent with similar instances in the past.
 
-## What can I do while tournament-banned?
+## What can I do while tournament-banned? {#activities}
```
Comment (Walavouchey) [inline wiki/Help_centre/Tournament_bans/en.md:39]:
```suggestion
## What can I do while tournament-banned? {#while-banned}
```

File: wiki/Help_centre/Website/en.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ tags:
 
 This section is dedicated to issues with the osu! website.
 
-## Common issues
+## Common issues {#common-issues}
 
-### I have not received a reply to my support ticket and it's been over 2 weeks!
+### I have not received a reply to my support ticket and it's been over 2 weeks! {#no-reply}
```
Comment (Walavouchey) [inline wiki/Help_centre/Website/en.md:23]:
```suggestion
### I have not received a reply to my support ticket and it's been over 2 weeks! {#support-no-reply}
```

File: wiki/Help_centre/Website/en.md [hunk 3]
```diff
@@ -18,23 +18,23 @@ tags:
...
 
-### Can I completely block another user from contacting me?
+### Can I completely block another user from contacting me? {#ignore-user}
 
 **Yes, absolutely.**
 
 In the event that another user is harassing you via private messages either on the forums or via the game client itself, there are a few steps that you can take yourself to filter out their messages.
 
-#### Blocking a user from contacting you in-game
+#### Blocking a user from contacting you in-game {#ignore-user-ingame}
```
Comment (Walavouchey) [inline wiki/Help_centre/Website/en.md:37]:
```suggestion
#### Blocking a user from contacting you in-game {#ignore-user-in-game}
```

File: wiki/Help_centre/Website/en.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ You'll no longer see public chat messages from any users on that list, or receiv
 
 If you wish to ignore highlights, private messages, or public messages, you can simply append respectively `@h`, `@p`, `@c` to their username. `The_Gatekeeper@c` will ignore the user's messages in public channels. It is possible to combine those: `The_Gatekeeper@ph` will ignore highlights and private messages, but leave public messages visible.
 
-#### Blocking all non-friends from sending private messages in-game
+#### Blocking all non-friends from sending private messages in-game {#ignore-strangers-ingame}
```
Comment (Walavouchey) [inline wiki/Help_centre/Website/en.md:49]:
```suggestion
#### Blocking all non-friends from sending private messages in-game {#ignore-strangers-in-game}
```

**Review by Walavouchey — DISMISSED** [review]

**Comment by Walavouchey** [comment]
conflicts

**Review by Walavouchey — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**